nautobot/dcim/models/devices.py

```diff
@@ -998,3 +1013,69 @@ def to_csv(self):
             self.domain,
             self.master.name if self.master else None,
         )
+
+
+@extras_features(
+    "custom_fields",
```

Comment:
Seeing as this is a `ConfigContextModel` does `config_context_owners` need to be included here in `extras_features`?

Reply:
`Device` doesn't either? 🤷